### DISCUSSION TITLE
export to PDF not working?

### DISCUSSION THREAD

**Discussion by roda69** [jgraph/drawio#2246] [Q&A] [answered]
Hi,

is there currently a known issue with exporting to PDF? I tried from the desktop app and got the spinning 'saving' icon, and tried from the web portal and got an http 500 error with url https://convert.diagrams.net/node/export. Just wondering if it had anything to do with the domain name change (draw.io to diagrams.net) ?

regards

Rod

**Comment by davidjgraph** [comment, marked as answer]
No, server problems, should be fixed now.

**Reply by roda69** [reply]
Excellent its working now - thanks @davidjgraph :) !

**Comment by aporreca10** [comment]
Hello, 
I am receiving errors when I try to export to PDF either saving the file of just downloading it. https://convert.diagrams.net/node/export
Error!

Thank you, 
Andrea
